package.json

```diff
@@ -134,7 +135,6 @@
     "lint:css": "stylelint --fix '**/*.scss'",
     "lint:js": "eslint . --fix --ext .js,.mdx,.ts,.tsx",
     "test": "cypress run",
-    "test:open": "cypress open",
-    "prepublishOnly": "npm ci && npm run check-lint && npm run preprocess && npm run build"
```

Comment:
Don't we still want the prepublish step for manual publishing?

Reply:
This npm script causes changesets to fail. I can add these steps to the manual publish documentation

Reply:
Cool, that works for me.

Reply:
Added ✅